### ISSUE TITLE
Interactivity mode give an ts error when use semicolon on property "mode"

### ISSUE THREAD

**Issue by davulrich** [Gamote/lottie-react#37]
**Describe the bug**
When you define interactivity const with property mode ('scroll' or 'cursor'), typescript display an error on Lottie component property interactivity - "Type 'string' is not assignable to type '"cursor" | "scroll"'."

**Expected behavior**
TS shouldn't display and error. Because the string is typed correctly.

**Screenshots**
![image](https://user-images.githubusercontent.com/46351885/133453642-b7a51a1e-e83c-4d90-891a-21c39dab1391.png)
![image](https://user-images.githubusercontent.com/46351885/133453909-b29a375a-5969-404a-ad0e-05355e24305e.png)

**Desktop (please complete the following information):**
 - OS: macOS BigSur
 - Editor: VS Code

**Comment by davulrich** [comment]
I temporary solved it by creating own interface like this:

```js
import Lottie, {Action} from 'lottie-react'

interface Interactivity {
  mode: 'scroll' | 'cursor'
  actions: Action[]
}

const interactivity: Interactivity = {
  mode: 'scroll',
  actions: [
    {visibility: [0, 0.1], type: 'stop', frames: [1]},
    {
      visibility: [0.1, 0.9],
      type: 'play',
      frames: [150],
    },
    {visibility: [0.9, 1], type: 'stop', frames: [1]},
  ],
}
```

**Comment by Gamote** [comment]
Hey @davulrich thank you for taking the time to report this. Using your configuration I am not able to recreate the error. Do you still encounter this issue?

**Comment by davulrich** [comment]
> 

Hi @Gamote , I wrongly explained in the description so I will try it now better again.

I need to assign a correct type to interactivity constant or TS will yell at me that interactivity prop is wrongly typed.

So I tried to import type InteractivityProps from lottie-react:
```js
import Lottie, {InteractivityProps} from 'lottie-react'

const interactivity: InteractivityProps = {...}
```
And TS showed me an error, but in docs there is no info regarding lottieObj property.
![image](https://user-images.githubusercontent.com/46351885/149928025-e69003d1-0543-4fee-9ea2-f6ea2d8b6f44.png)

So I solved it by creating my own custom interface:
```js
import Lottie, {Action} from 'lottie-react'

interface Interactivity {
  mode: 'scroll' | 'cursor'
  actions: Action[]
}

const interactivity: Interactivity = {
```
Conclusion - It would be great to be able to import InteractiveProps from lottie react and don't bother with custom typing.
